app/AppComponents.scala

```diff
@@ -0,0 +1,45 @@
+import com.amazonaws.auth.profile.ProfileCredentialsProvider
+import com.amazonaws.auth.{ InstanceProfileCredentialsProvider, SystemPropertiesCredentialsProvider, EnvironmentVariableCredentialsProvider, AWSCredentialsProviderChain }
+import com.amazonaws.regions.{ Region, Regions }
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBAsyncClient
+import controllers._
+import play.api.ApplicationLoader.Context
+import play.api.i18n.{ DefaultLangs, DefaultMessagesApi, MessagesApi }
+import play.api.libs.ws.ahc.AhcWSComponents
+import play.api.{ BuiltInComponents, BuiltInComponentsFromContext }
+import router.Routes
+import play.api.routing.Router
+
+trait AWSComponent { self: BuiltInComponents =>
+  private val awsCreds = new AWSCredentialsProviderChain(
+    new EnvironmentVariableCredentialsProvider(),
```

Comment:
Can this just be: 

`new EnvironmentVariableCredentialsProvider(),
new ProfileCredentialsProvider("developerPlayground"),
new InstanceProfileCredentialsProvider()`